pkgs/development/python-modules/av/default.nix

```diff
@@ -38,6 +44,12 @@ buildPythonPackage rec {
 
   buildInputs = [ ffmpeg-headless ];
 
+  env = {
```

Comment:
Actually they dropped this recently https://github.com/PyAV-Org/PyAV/commit/2186a380ef7d7b0dcc3427d0208f31edccc821ea

Reply:
Opened https://github.com/NixOS/nixpkgs/pull/440158.